packages/vats/decentral-devnet-config.json

```diff
@@ -114,21 +159,27 @@
     }
   },
   "bundles": {
```

Comment:
Not crucial, but I would prefer alphabetizing these properties instead of changing between two randomish orderings.